packaging/aznfs/RPM/aznfs.spec

```diff
@@ -78,6 +86,9 @@ if [ $1 == 0 ]; then
 		echo
 		if [ "$result" != "y" -a "$result" != "Y" ]; then
 			echo "Removal aborted!"
+			echo -e "${RED} Dependencies may have uninstalled required for ANZFS package."
+			echo -e "Please check the required dependencies removed as part of this removal process and install it manually."
+			echo -e "Please contact Microsoft Support if not able to resolve required dependencies.${NORMAL}"
```

Comment:
echo
echo `"*******************************************************************"`
echo "Unfortunately some of the anzfs dependencies may have been uninstalled."
echo "To fix this you may re-install the latest aznfs package, o/w existing"
echo "aznfs mounts may be affected and new aznfs shares cannot be mounted."
echo `"*******************************************************************"`
echo